pkg/cache/cache_api.go

```diff
@@ -73,7 +70,7 @@ type ModelCache interface {
 	// Returns:
```

Comment:
Will we have other informations in future? in that case, we may need complex structure as well?